src/core_modules/capture-core/components/Pages/EditEvent/DataEntry/epics/editEventDataEntry.epics.js

```diff
@@ -56,43 +49,28 @@ const runRulesForEditSingleEvent = (store: ReduxStore, dataEntryId: string, item
     const state = store.value;
     const formId = getDataEntryKey(dataEntryId, itemId);
     const eventId = state.dataEntries[dataEntryId].eventId;
-    const event = state.events[eventId];
-    const metadataContainer = getProgramAndStageFromEvent(event);
+    const { programId } = state.currentSelections;
+    const eventProgram = getEventProgramThrowIfNotFound(programId);
```

Comment:
Now this can throw an error that will basically break the app. Do we want the app to break or would it be more graceful to catch the error in the epics below?

Reply:
It's ok to break here, because this should never ever happen.

Reply:
I agree it should never happen. I was just thinking perhaps in the future we inform the user why this happened somehow that completely breaking the experience for them